docs/conf.py

```diff
@@ -296,14 +297,13 @@
 ]
 
 # Documents to append as an appendix to all manuals.
-#texinfo_appendices = []
+# exinfo_appendices = []
 
 # If false, no module index is generated.
-#texinfo_domain_indices = True
+# exinfo_domain_indices = True
 
 # How to display URL addresses: 'footnote', 'no', or 'inline'.
-#texinfo_show_urls = 'footnote'
+# exinfo_show_urls = 'footnote'
```

Comment:
@shreyans800755 @jayvdb @sils Exactly the latter suggestion... Just a little mistake in my replace regex :D Sorry... Will correct now!